supplementary_style_guide/glossary_terms_conventions/product_conventions/openshift.adoc

```diff
@@ -553,6 +688,17 @@ _Update the `Pod` spec to reflect the changes._
 
 *See also*:
 
+[discrete]
+[[storageclass]]
+==== StorageClass (noun)
```

Comment:
Is "StorageClass" really correct? In core OCP, we would refer to this as "storage class", and a quick spot check in the OCS docs seemed to do the same.